meson.build

```diff
@@ -40,3 +40,18 @@ subdir('src')
 
 # Build the binaries.
 subdir('cmdline')
+
+version = run_command('git', 'describe', '--tags')
+if version.returncode() != 0
+  version = 'unknown-error'
```

Comment:
If you're building without git maybe it would be better not to build the pkg-config file at all?  Otherwise this might get somewhat confusing.  OTOH debugging why you can't build the pkg-config file might also be annoying, maybe going `message('git version not found, pkg-config will not be generated')` is the right thing to do then?  What do you think?